samples/Makefile

```diff
@@ -12,15 +12,19 @@ SYS	= c64
 # source tree; otherwise, use the "install" directories.
 ifeq "$(wildcard ../src)" ""
 # No source tree
-MOUS = /usr/lib/cc65/mou/$(SYS)*.mou
-TGI  = /usr/lib/cc65/tgi/$(SYS)*.tgi
+MOUS = /usr/lib/cc65/target/$(SYS)/drv/mou/$(SYS)*.mou
```

Comment:
There are two redundant pieces in that pathname.  I made a comment about the `mou/` repetition, elsewhere.  `$(SYS)`, also, is repeated, needlessly.  A shorter path filter is
```
MOUS = /usr/lib/cc65/target/$(SYS)/drv/*.mou
```